src/main/webapp/js/instructor.js

```diff
@@ -434,3 +434,35 @@ function updateHoverShowPictureEvents(actualLink, resolvedLink) {
 }
 
 // --------------------------------------------------------------------------
+
+/**
+ * Selects contents inside an element.
+ * @param elementNode a HTML DOM element object, not a jQuery object
```

Comment:
Would be good to mention the purpose of `elementNode` in the context of the method as well. Also it should be sufficient to just mention that it is a HTML DOM object, not necessary to mention that it isn't a jQuery object.
i.e. `elementNode The HTML DOM object to select contents from.`